reportportal/values.yaml

```diff
@@ -5,6 +5,18 @@
 ## String to fully override reportportal.fullname template
 ##
 # fullnameOverride:
+extraInitContainers: {}
+
+# Extra init containers to e.g. wait for minio
+#  - name: "wait-for-minio"
+#    image: "busybox"
+#    imagePullPolicy: "IfNotPresent"
+#    securityContext:
+#      runAsNonRoot: true
+#    command:
+#      - sh
+#      - "-c"
+#      - "for i in {1..100}; do sleep 1; if nc -vz <minio-endpoint> <minio-port>; then exit 0; fi; done; exit 1"
```

Comment:
I tried to use it putting my own values: `for i in {1..100}; do sleep 1; if nc -vz minio.default.svc.cluster.local 9000; then exit 0; fi; done; exit 1` and it does not work for me. Looks like there is an issue with the example. Sorry, I can't let it go because of that. Even if example is commented, we have to put only correct examples here.

Reply:
Hi @HardNorth ,

Sorry, there was a mistake in the example , i just fixed it : https://github.com/reportportal/kubernetes/pull/233/files#diff-c554617a65c1ad51602ac8780bbe79db48ddcc7b00106d755361a16301b571ec